### ISSUE TITLE
some question

### ISSUE THREAD

**Issue by junkangwu** [xhuang31/GraphRNA_KDD19#1]
你好，请问github上这份graphRNA源码上是没有采用k-fold cross-validation吗？ 我在学习论文以及代码的过程中，产生了这个疑惑。望解答谢谢！

**Comment by xhuang31** [comment]
Thanks for your interest. 
In the experiments, I used five-fold cross-validation.
In the code, I only illustrated the evaluation process of one case, i.e., Group2 >=21. There are four other cases, including [1 to 5], [6 to 10], [11 to 15], and [16 to 20].

And in the experiments, I repeated the above five cases for ten times, with different random shuffle. 

Thanks.

**Comment by junkangwu** [comment]
Thanks for your detailed explaination! 
Thanks!

**Comment by junkangwu** [comment]
Sorry, I have one more question：
In your figure 1 in 2019KDD-GraphRNA method paper,  δ means the jth node attribute category. I'm confused about that why node 1 has δ1,δ2,δ3? Does that mean node one and  1th/2rd/3st node have the same attribute category?

**Comment by xhuang31** [comment]
Let's assume that δ1, δ2, δ3, δ4 denotes the words "one" "two" "three" and "four".

Then figure 1 means that, node 1 has words "one" "two" "three" in its node attributes. 
Node 2 only has word "two" in its node attributes. 
Node 3 has words "two" and "four" in its node attributes. 

Thanks.

**Comment by junkangwu** [comment]
Thanks again！

**Comment by junkangwu** [comment]
Hi here！I'm sorry to bother you again.
In your paper, you say `GCN` is a special case of your model. But when I use your code runing on some smaller dataset such as `cora, pubmed`. The performance of graphRNA is not good. I found that the model was overfitted very early in the experiment. May I ask your help about how to deal with this kind of problem? I want to obtain similar performance as `GCN`.
Of course,I reduced the model to vanilla GCN according to the method in your paper(removing the bidirectional RNN layer.

**Comment by xhuang31** [comment]
It is a hyper-parameters tuning problem. For example, there are many ways to handle overfitting, including reducing the layers, early stopping, using smaller models.

Thanks.
